packages/mui-base/src/Slider/Slider.tsx

```diff
@@ -126,6 +126,7 @@ const Slider = React.forwardRef(function Slider<RootComponentType extends React.
     marked: marks.length > 0 && marks.some((mark) => mark.label),
     dragging,
     focusedThumbIndex,
+    activeThumbIndex: active,
```

Comment:
@michaldudak similarly to `focusedThumbIndex`, I added the `activeThumbIndex` to allow custom component utilizing this information. I know we discussed something similar in the past, but I really feel like this calculated feels should be accessible in the `slotProps` callback. Maybe we can extend it so that apart from the `ownerState`, we have two additional states in the callback - `active` and `focused` so that we don't need to create custom components just for this calculation. We decided in the past that we should not include them in the `ownerState`, as they are not really a state of the owner component, but still having access to them feels important.

Reply:
From a developer's perspective, yeah, it is totally understandable that someone would expect to have an easy way of applying props based on the active state. Having them as additional parameters of the slotProps callback feels like the best way to achieve this. This way we won't pollute the `ownerState` with something that does not belong to the owner.
Additionally, for styling, we could make sure that the built-in CSS pseudoclasses (`:active` and `:focus-visible`) can be used. For now, I see that only `:active` works well.

Reply:
> From a developer's perspective, yeah, it is totally understandable that someone would expect to have an easy way of applying props based on the active state. Having them as additional parameters of the slotProps callback feels like the best way to achieve this. This way we won't pollute the ownerState with something that does not belong to the owner.

Perfect, I will create a pull request for this soon.

> Additionally, for styling, we could make sure that the built-in CSS pseudoclasses (:active and :focus-visible) can be used. For now, I see that only :active works well.

Yep

Reply:
Here it is https://github.com/mui/material-ui/pull/37749